net/lftp/Makefile

```diff
@@ -8,13 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lftp
-PKG_VERSION:=4.7.5
+PKG_VERSION:=4.8.2
 PKG_RELEASE:=1
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://lftp.tech/ftp/ \
```

Comment:
The ftp mirror is dead, please add another mirror.
Something like: https://mirror.csclub.uwaterloo.ca/gentoo-distfiles/distfiles/ as primary (first) and http://lftp.tech/ftp/ as secondary. Also, please add a comment https://github.com/lede-project/source/pull/1133 if you want an alias and better overall availability (it'd add more mirrors).

Reply:
The layout of the description seems right, what's wrong with it ?
![image](https://user-images.githubusercontent.com/8364158/31067822-2d4a19c0-a755-11e7-8a1f-bf3ca25f5c11.png)